config.d.ts

```diff
@@ -16,10 +16,17 @@ declare module '*.yaml' {
     path: string;
   }
 
+  interface AboutUs {
+    text: string;
+    image: string;
+    altText: string;
+  }
+
   interface Config {
     pageInfo: PageInfo[];
     bannerInfo: BannerInfo;
     heroIcons: HeroIcon[];
+    aboutUs: AboutUs;
```

Comment:
Thoughts on changing this to like:
```
interface HomeSection {
  title: string;
  content: string;
  image: string;
  imageAlt: string;
  buttonRoute?: string;
}
```
 Then we could automatically add more sections to the page easily in terms of text content, I know this doesnt take into consideration other section types but I think thats okay, If you want to keep it the way it is thats fine too, I just figure making it a little more open makes things more expandable in the future.

Reply:
I like that idea. not 100% sure how it should be implemented in the main page (like if we want to do stuff back to back or grab specific indexes to cherrypick but open to discussion on that. For now that feature is implimented :)